package.json

```diff
@@ -3,15 +3,25 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "start": "node ./bin/www"
+    "start": "node ./bin/www",
+    "setup": "npm run setup:db",
+    "setup:db": "bash db/setupDevelopmentDb.sh",
```

Comment:
Curious why a setup local db script needs to be here if docker-compose will have the db